cpg_workflows/targets.py

```diff
@@ -512,7 +532,7 @@ def __init__(
         self.alignment_input_by_seq_type: dict[str, AlignmentInput] = (
             alignment_input_by_seq_type or dict()
         )
-        self.seq_by_type: dict[str, Sequence] = seq_by_type or dict()
+        self.assays: dict[str, Assay] = assays or dict()
```

Comment:
Not a big deal, but I thought pylint would complain about this
```suggestion
        self.assays: dict[str, Assay] = assays or {}
```